Source/player.cpp

```diff
@@ -1683,152 +1683,109 @@ void __fastcall RespawnDeadItem(ItemStruct *itm, int x, int y)
 
 void __fastcall StartPlayerKill(int pnum, int earflag)
 {
-	unsigned int v2; // edi
-	unsigned int v3; // esi
-	char v4; // al
-	int v5; // ecx
-	int v6; // ST0C_4
-	bool v7; // zf
-	int *v8; // eax
-	signed int v9; // ecx
-	char *v10; // eax
-	char v11; // al
-	short v12; // cx
-	short v13; // ax
-	int v14; // ecx
-	int v15; // eax
-	signed int v17; // ebx
-	int v18; // eax
-	ItemStruct ear; // [esp+Ch] [ebp-178h]
-	BOOL v20; // [esp+17Ch] [ebp-8h]
-	struct ItemStruct *itm; // [esp+180h] [ebp-4h]
-
-	v2 = pnum;
-	v3 = 21720 * pnum;
-	itm = (struct ItemStruct *)earflag;
-	if ( plr[pnum]._pHitPoints <= 0 && plr[pnum]._pmode == PM_DEATH )
+	if ( plr[pnum]._pHitPoints <= 0 && plr[pnum]._pmode == PM_DEATH ) {
 		return;
-	if ( myplr == pnum )
-		NetSendCmdParam1(1u, CMD_PLRDEAD, earflag);
-	v20 = (unsigned char)gbMaxPlayers > 1u && plr[v3 / 0x54D8].plrlevel == 16;
-	if ( v2 >= 4 )
-		TermMsg("StartPlayerKill: illegal player %d", v2);
-	v4 = plr[v3 / 0x54D8]._pClass;
-	if ( v4 )
-	{
-		if ( v4 == 1 )
-		{
-			v5 = PS_ROGUE71;
-		}
-		else
-		{
-			if ( v4 != 2 )
-				goto LABEL_18;
-			v5 = PS_MAGE71;
-		}
-		PlaySfxLoc(v5, plr[v3 / 0x54D8].WorldX, plr[v3 / 0x54D8].WorldY);
-		goto LABEL_18;
 	}
-	PlaySfxLoc(PS_DEAD, plr[v3 / 0x54D8].WorldX, plr[v3 / 0x54D8].WorldY); /// BUGFIX: should use `PS_WARR71` like other classes
-LABEL_18:
-	if ( plr[v3 / 0x54D8]._pgfxnum )
-	{
-		plr[v3 / 0x54D8]._pgfxnum = 0;
-		plr[v3 / 0x54D8]._pGFXLoad = 0;
-		SetPlrAnims(v2);
+
+	if ( myplr == pnum ) {
+		NetSendCmdParam1(TRUE, CMD_PLRDEAD, earflag);//23

```

Comment:
Remove `//23` comment, or provide rationale for keeping it :)

Reply:
Oops that is me adding ASM line numbers for the function when I try to match things up :)